flow/envs/base.py

```diff
@@ -670,6 +677,18 @@ def terminate(self):
             # close pyglet renderer
             if self.sim_params.render in ['gray', 'dgray', 'rgb', 'drgb']:
                 self.renderer.close()
+            # generate video
+            elif (self.sim_params.render is True) and self.sim_params.save_render:
+                save_dir = os.path.expanduser('~') + '/flow_movies'
+                images_dir = self.path.split('/')[-1]
+                if not os.path.exists(save_dir):
+                    os.mkdir(save_dir)
+                speedup = 10  # speedup multiplier
+                fps = speedup//self.sim_step
+                os_cmd = "ffmpeg -y -r {fps} -i {path}/frame_%06d.png".format(path=self.path, fps=fps)
+                os_cmd += " -pix_fmt yuv420p " + self.path+"/%s.mp4" % images_dir
+                os_cmd += "&& cp " + self.path+"/%s.mp4 " % images_dir + save_dir + "/"
+                os.system(os_cmd)  # only works in Unix
```

Comment:
You can subprocess.Popen for a command that works in both unix and mac

Reply:
I actually didn't want to follow this pattern that I saw in `visuallizer_rllib.py` (i was assuming that's how you guys were using this). If we merge flow_movies with flow_rendering, subprocess.Popen can be used. (subprocess somehow cannot be configured to wait for ffmpeg to finish running, hence chaining a copy command wouldn't work.)

Reply:
https://stackoverflow.com/questions/15107714/wait-process-until-all-subprocess-finish

Reply:
Last thing; can we clean and remove the screenshots after this command completes?

Reply:
Yea, i did check out wait, but it doesn't work for ffmpeg for some strange reason. Yes, we can clean up all auxiliary files. I'm adding that in.